### ISSUE TITLE
Grouping brings to front

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-paint#93]
Grouping should try to maintain layer order

**Comment by towerofnix** [comment]
I agree, but it should be noted that the Scratch 2.0 vector editor does bring grouped items to the front layer.

**Comment by fsih** [comment]
Good point, thanks for the info!
